dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/dto/DatasetDTO.java

```diff
@@ -17,7 +17,12 @@
     private String globalIdCreateTime;
     private String publisher;
     private String publicationDate;
-    private boolean isGuestbookAvailable;
+
+    /**
+     * Flags whether there is an active guestbook on dataset.
+     * Need for OpenAire exporter <rightsList> element specification.
+     */
+    private boolean hasActiveGuestbook;
```

Comment:
We have convention that javadocs are above methods :P, (getter in this case).

Reply:
We do? :D

Reply:
That's what I heard some time ago :P.